codex-ui/dev/pages/components/Picture.vue

```diff
@@ -21,7 +21,7 @@
 import PageHeader from '../../components/PageHeader.vue';
 import { Picture } from '../../../src/vue';
 
-const pictureNames = ['Hammer', 'DarkTheme', 'LightTheme', 'SystemTheme'];
+const pictureNames = ['Hammer', 'DarkTheme', 'LightTheme', 'SystemTheme', 'NotexLogo'];
```

Comment:
1. logo should be moved to the NoteX component instead of CodeX UI. As well as Hummer.
2. It's better to separate 'DarkTheme', 'LightTheme', 'SystemTheme` for own components